### PR TITLE
OCPBUGS-41852: add tested IBMCloud instance types in 4.17 test

### PR DESCRIPTION
**Pull request by MayXuQQ**
add new vm type found in 4.17 test.
[OCPBUGS-41852](https://issues.redhat.com/browse/OCPBUGS-41852)

### CODE REVIEW

**Comment by MayXuQQ** [comment]
@sadasu could you help review it, thanks a lot

File: docs/user/ibmcloud/tested_instance_types_x86_64.md [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
 * `cx2-8x16`
 * `cx2d-4x8`
 * `cx3d-8x20`
+* `cx3dc-4x10`
 * `gx2-8x64x1v100`
 * `gx3-16x80x1l4`
 * `mx2-8x64`
 * `mx2d-4x32`
-* `mx3d-2x20`
-* `ox2-4x32`
```
Comment (barbacbd) [inline docs/user/ibmcloud/tested_instance_types_x86_64.md:12]: Do we no longer support these types?

Comment (MayXuQQ) [reply docs/user/ibmcloud/tested_instance_types_x86_64.md:12]: replace  `mx3d-2x20` with 'mx3d-4x40 '
replace `ox2-4x32` with `ox2-8x64`
1. list one type in the same profiles
2. use the larger to support the master and workers.

**Comment by MayXuQQ** [comment]
@sadasu could you give a lgtm? thanks

**Pull request merged**